src/util/twemojify.jsx

```diff
@@ -42,7 +56,15 @@ export function twemojify(text, opts, linkify = false, sanitize = true, maths =
   if (sanitize) {
     content = sanitizeText(content);
   }
+  
+  if(opts){
+    opts.base = getTwemojiCDN()
```

Comment:
It seems like it always override the `opts.base` option. It should only set it if `opts.base` is `undefined`.